tests/test_gradient.py

```diff
@@ -15,6 +15,7 @@
 class TestGradient(object):
 
     @skipif(['chkpnt', 'cpu64-icc'])
+    @switchconfig(safe_math=True)
```

Comment:
is it correct to say that with the new interpolation loop nest structure we're somehow losing some accuracy?

how significant is the failure without safe_math?

Reply:
It's quite small, about 1e-5.